source/internal/_createPartialApplicator.js

```diff
@@ -1,11 +0,0 @@
-import _arity from './_arity';
-import _curry2 from './_curry2';
-
-
-export default function _createPartialApplicator(concat) {
-  return _curry2(function(fn, args) {
-    return _arity(Math.max(0, fn.length - args.length), function() {
-      return fn.apply(this, concat(args, arguments));
-    });
-  });
-}
```

Comment:
I do love getting rid of many `_internal` functions.  They've always rubbed me the wrong way.